model_transactions.go

```diff
@@ -157,11 +157,47 @@ func (m *Transaction) setXPubID() {
 	}
 }
 
+// getTransactions will get all the transactions with the given conditions
+func getTransactions(ctx context.Context, metadata *Metadata, conditions *map[string]interface{},
+	queryParams *datastore.QueryParams, opts ...ModelOps) ([]*Transaction, error) {
+
+	modelItems := make([]*Transaction, 0)
+	if err := getModelsByConditions(ctx, ModelTransaction, &modelItems, metadata, conditions, queryParams, opts); err != nil {
+		return nil, err
```

Comment:
Does opts need the `...`

`opts` vs `opts...`

Not sure if that is an error or not...